.travis.yml

```diff
@@ -11,12 +11,16 @@ matrix:
 install:
 - |
   if [[ ${MASON_PLATFORM:-unset} == 'unset' ]] || [[ ${MASON_PLATFORM} == 'osx' ]] || [[ ${MASON_PLATFORM} == 'linux' ]]; then
-    echo "Installing clang for OSX/Linux via 'install' target"
-    source utils/toolchain.sh
-    which ${CC}
-    ${CC} -v
-    which ${CXX}
-    ${CXX} -v
+    # If platform is Linux or OSX, we might still be cross compiling if the
+    # MASON_PLATFORM_VERSION is not the same as the host platform.
+    if [[ ${MASON_PLATFORM_VERSION} == `uname -m` ]] || [[ -z ${MASON_PLATFORM_VERSION} ]]; then
```

Comment:
But when building for Cortex A9, `MASON_PLATFORM` will be set to `linux`, but `MASON_PLATFORM_VERSION` will be `cortex_a9`.